### DISCUSSION TITLE
FOV in décimal degré, but too in Minute & Second

### DISCUSSION THREAD

**Discussion by proceau** [Stellarium/stellarium#802] [Q&A] [answered]
On FOV, print Decimal Degré, but add Degre, minute, second version, because 0,0033 it's less clean than 0° 0' 12"

I'm search on google, parameter, not find any solution, sorry if exist.

**Comment by gzotti** [comment]
There is a global switch for decimal degrees vs. DMS, but FoV has always been displayed in decimal degrees. Should the global switch also control this, or shall we use yet another config option?

**Comment by alex-w** [comment, marked as answer]
The config option to switch FOV in DMS format implemented a long time ago, but who read User Guide?

Hint: gui/flag_fov_dms = true

**Comment by proceau** [comment]
I'm read and search on user guide, by fov searchkey but not find :/
on google flag_fov_dms only 11 result, but very thx all !

**Comment by alex-w** [comment]
> I'm read and search on user guide, by fov searchkey but not find :/
> on google flag_fov_dms only 11 result, but very thx all !

See page 295 of Stellarium 0.19.2 User Guide
